sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/StringFormatValidator.cs

```diff
@@ -86,6 +85,9 @@ public class StringFormatValidator : SonarDiagnosticAnalyzer
                 ValidationFailure.UnusedFormatArguments
             };
 
+        private static readonly Regex StringFormatItemRegex = // pattern is {index[,alignment][:formatString]}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS125)